Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
Hmmm we can do that. I think it's still better to have a TimedVM protocol that extends BaseVM though, but the difference is that it includes the timer. Because `updateTime` should be in TimedVM right not BaseVM?